### PR TITLE
update error handling when not connected to lsp

### LINKED ISSUE #1176: Bad UI for Breez offline
1. Turn off WiFi and Mobile data on the device.
2. Click Receive>Receive via Invoice
3. Enter the amount of sats
4. Click Create

image

NOTE: The flush bar that says Breez is offline appears properly
@kingonly please advise on the proper outcome for this one.

### PR DESCRIPTION
**Pull request by ubbabeck**
fixes #1176 

If the returned string indicates that there is no internet connection, we'll display the following error both for lnurl and create invoice.

https://github.com/breez/breezmobile/assets/36157890/9607e918-6bab-4b6c-bd0c-7d76f363a5ac

### CODE REVIEW

File: lib/routes/create_invoice/create_invoice_page.dart [hunk 1]
```diff
@@ -154,7 +155,8 @@ class CreateInvoicePageState extends State<CreateInvoicePage> {
                             var loaderRoute = createLoaderRoute(context);
                             try {
                               navigator.push(loaderRoute);
-
+                              log.info(
+                                  "lsp status ${lspStatus.lastConnectionError}");
```
Comment (erdemyerebasmaz) [inline lib/routes/create_invoice/create_invoice_page.dart:159]: parenthesis after the quote

File: lib/routes/create_invoice/create_invoice_page.dart [hunk 2]
```diff
@@ -187,8 +189,9 @@ class CreateInvoicePageState extends State<CreateInvoicePage> {
                                   }
                                   showFlushbar(
                                     context,
-                                    message: texts
-                                        .qr_code_dialog_error(e.toString()),
+                                    message: texts.qr_code_dialog_error(
+                                        extractExceptionMessage(e,
+                                            texts: texts)),
```
Comment (erdemyerebasmaz) [inline lib/routes/create_invoice/create_invoice_page.dart:194]: comma between parenthesis

File: lib/routes/create_invoice/create_invoice_page.dart (hunk 1 above)
Comment (erdemyerebasmaz) [inline lib/routes/create_invoice/create_invoice_page.dart:159]: comma after string

**Review by erdemyerebasmaz — APPROVED** [review]
LGTM
Two nit comments regarding formatting & indentation

File: lib/routes/create_invoice/create_invoice_page.dart (hunk 1 above)
Comment (ubbabeck) [reply lib/routes/create_invoice/create_invoice_page.dart]: Added

File: lib/routes/create_invoice/create_invoice_page.dart (hunk 2 above)
Comment (ubbabeck) [reply lib/routes/create_invoice/create_invoice_page.dart]: Added

**Pull request merged**